cogs/login.py

```diff
@@ -0,0 +1,146 @@
+import logging
+from typing import Any, Dict, cast
+
+import discord
+from discord.ui import Select, View
+from discord import Color, Embed, Guild
+from discord.ext import commands
+from discord.ext.commands.context import Context
+
+
+from main import ManChanBot
+
+from .commandbase import CommandBase
+
+import json
+
+with open("login.json") as f:
```

Comment:
This will set the json load to be global. If we change a setting in the login.json then it will not be reflected.

Move this to be a method call that opens and read the login info

Reply:
Done, just did:

```py
def parse_login():
    with open("login.json", "r") as f:
        return json.load(f)
```

then called `config = parse_json()` in the login function